profiling/src/php_ffi.h

```diff
@@ -120,3 +120,10 @@ void ddog_php_prof_zend_mm_set_custom_handlers(zend_mm_heap *heap,
                                                void* (*_realloc)(void*, size_t));
 
 zend_execute_data* ddog_php_prof_get_current_execute_data();
+
+/**
+ * The following two functions exist for the sole purpose of creating fake stack
+ * frames that can be used in testing/benchmarking scenarios
+ */
+zend_execute_data* ddog_php_test_create_fake_zend_execute_data(int depth);
```

Comment:
Not too familiar with codebase, though I'm guessing it could be better to create a dedicated ffi header ?